packages/platform-sdk-profiles/src/plugins/plugin-repository.test.ts

```diff
@@ -52,22 +52,11 @@ it("should throw if a plugin cannot be found by its ID", () => {
 });
 
 it("should restore previously created data", () => {
-	subject.fill({ data: { [stubPlugin.id]: stubPlugin }, blacklist: [] });
+	subject.fill({ data: { [stubPlugin.id]: stubPlugin } });
```

Comment:
This can now be a flat object, no need for a separate `data` key because we won't store anything else here.

Reply:
Missed it. Thanks